data/src/emulator.js

```diff
@@ -3986,14 +3986,19 @@ class EmulatorJS {
             }
         } else if (option === "vsync") {
             this.gameManager.setVSync(value === "enabled");
+        } else if (option === "videoRotation") {
+            this.gameManager.setVideoRotation(value);
         }
     }
     menuOptionChanged(option, value) {
         this.saveSettings();
         if (this.debug) console.log(option, value);
         if (!this.gameManager) return;
         this.handleSpecialOptions(option, value);
-        this.gameManager.setVariable(option, value);
+        const notCoreOption = [ "webgl2Enabled", "videoRotation"];
```

Comment:
What is this for

Reply:
There is no need to send invalid core options to retroarch

Reply:
I don't think we should check for this, There are many more "invalid" core options than just these 2, and we're trying to move away from magic variables. The setVariable function also properly handles unknown core options